applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -76,6 +76,7 @@ struct temp_cap_storage {
 	struct bt_conn *conn;
 	/* Must be the same size as sink_codec_cap and source_codec_cap */
 	struct bt_codec codec[CONFIG_CODEC_CAP_COUNT_MAX];
+	uint8_t num_caps;
```

Comment:
Ref. the comment on line 802-804: We can make this clearer by 
```suggestion
	uint8_t num_caps_total;
```

Reply:
Good point, done